helper/resource/importstate/examplecloud_test.go

```diff
@@ -188,3 +231,251 @@ func examplecloudZoneRecord() testprovider.Resource {
+							tftypes.NewValue(tftypes.String, "fast")}),
+					"magic_numbers": tftypes.NewValue(
+						tftypes.List{ElementType: tftypes.Number}, []tftypes.Value{
+							tftypes.NewValue(tftypes.Number, 5),
+							tftypes.NewValue(tftypes.Number, 2)}),
+					"beep_boop": tftypes.NewValue(
+						tftypes.List{ElementType: tftypes.Bool}, []tftypes.Value{
+							tftypes.NewValue(tftypes.Bool, false),
+							tftypes.NewValue(tftypes.Bool, true),
+						}),
+				},
+			)),
+		},
+		ImportStateResponse: &resource.ImportStateResponse{
+			State: tftypes.NewValue(
+				tftypes.Object{
+					AttributeTypes: map[string]tftypes.Type{
+						"hostname":      tftypes.String,
+						"cabinet":       tftypes.String,
+						"unit":          tftypes.Number,
+						"active":        tftypes.Bool,
+						"tags":          tftypes.List{ElementType: tftypes.String},
+						"magic_numbers": tftypes.List{ElementType: tftypes.Number},
+						"beep_boop":     tftypes.List{ElementType: tftypes.Bool},
+					},
+				},
+				map[string]tftypes.Value{
+					"hostname": tftypes.NewValue(tftypes.String, "mail.example.net"),
+					"cabinet":  tftypes.NewValue(tftypes.String, "A1"),
+					"unit":     tftypes.NewValue(tftypes.Number, 14),
+					"active":   tftypes.NewValue(tftypes.Bool, true),
+					"tags": tftypes.NewValue(
+						tftypes.List{ElementType: tftypes.String}, []tftypes.Value{
+							tftypes.NewValue(tftypes.String, "storage"),
+							tftypes.NewValue(tftypes.String, "fast")}),
+					"magic_numbers": tftypes.NewValue(
+						tftypes.List{ElementType: tftypes.Number}, []tftypes.Value{
+							tftypes.NewValue(tftypes.Number, 5),
+							tftypes.NewValue(tftypes.Number, 2)}),
+					"beep_boop": tftypes.NewValue(
+						tftypes.List{ElementType: tftypes.Bool}, []tftypes.Value{
+							tftypes.NewValue(tftypes.Bool, false),
+							tftypes.NewValue(tftypes.Bool, true),
+						}),
+				},
+			),
+			Identity: teststep.Pointer(tftypes.NewValue(
+				tftypes.Object{
+					AttributeTypes: map[string]tftypes.Type{
+						"cabinet":       tftypes.String,
+						"unit":          tftypes.Number,
+						"active":        tftypes.Bool,
+						"tags":          tftypes.List{ElementType: tftypes.String},
+						"magic_numbers": tftypes.List{ElementType: tftypes.Number},
+						"beep_boop":     tftypes.List{ElementType: tftypes.Bool},
+					},
+				},
+				map[string]tftypes.Value{
+					"cabinet": tftypes.NewValue(tftypes.String, "A1"),
+					"unit":    tftypes.NewValue(tftypes.Number, 14),
+					"active":  tftypes.NewValue(tftypes.Bool, true),
+					"tags": tftypes.NewValue(
+						tftypes.List{ElementType: tftypes.String}, []tftypes.Value{
+							tftypes.NewValue(tftypes.String, "storage"),
+							tftypes.NewValue(tftypes.String, "fast")}),
+					"magic_numbers": tftypes.NewValue(
+						tftypes.List{ElementType: tftypes.Number}, []tftypes.Value{
+							tftypes.NewValue(tftypes.Number, 5),
+							tftypes.NewValue(tftypes.Number, 2)}),
+					"beep_boop": tftypes.NewValue(
```

Comment:
Really really reaching for an list of booleans example 😃

Reply:
Love it 😆